schedule/sles4sap/sles4sap_gnome_saptune.yaml

```diff
@@ -8,7 +8,6 @@ vars:
   BOOT_HDD_IMAGE: '1'
   TEST_CONTEXT: 'OpenQA::Test::RunArgs'
   NODE_COUNT: '1'
-  QESAP_CONFIG_FILE: 'sles4sap_azure_generic.yaml'
```

Comment:
Has been moved to JobGroup YAML definition?

Reply:
> Has this variable been moved to JobGroup YAML definition?

Yes.